robot_skills/src/robot_skills/world_model_ed.py

```diff
@@ -127,7 +127,7 @@ def get_closest_room(self, center_point=None, radius=0):
 
         return self.get_closest_entity(type="room", center_point=center_point, radius=radius)
 
-    def get_closest_laser_entity(self, type="", center_point=kdl.Vector(), radius=0):
+    def get_closest_laser_entity(self, type="", center_point=VectorStamped(), radius=0):
```

Comment:
VectorStamped() defaults to 0,0,0,0,0,0 in /map AFAIK. 

It has to have a default because type also already has one.